packages/patternfly-3/patternfly-react/src/components/DualListSelector/components/DualListSort.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Icon } from '../../..';
+import { noop } from '../../../common/helpers';
+
+const DualListSort = ({ onClick, side, isSortAsc }) => (
+  <Icon
+    aria-label="sort-list"
```

Comment:
Should be straight text like `reverse sort order`, allow for override.

Reply:
renamed the aria label and added a prop to override.